boards/shields/adafruit_16_ch_pwm_servo/adafruit_16_ch_pwm_servo.overlay

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
I think shield directory should be just `boards/shields/pca9685`.

servo in `adafruit_16_ch_pwm_servo.overlay` is a bit confusing even it is the name, maybe it is better to call it `pca9658_generic_pwmled` shield?

Reply:
I've shortened the board name to `adafruit_pca9685`